superset/migrations/versions/2023-06-29_18-38_240d23c7f86f_update_tag_model_w_description.py

```diff
@@ -17,14 +17,14 @@
 """update_tag_model_w_description
 
 Revision ID: 240d23c7f86f
-Revises: 6fbe660cac39
+Revises: 8e5b0fb85b9a
```

Comment:
What's this fixing?

Reply:
We currently have 2 heads in Alembic, this chains the 2 (independent heads) so there's just one.